pom.xml

```
@@ -36,7 +36,7 @@
         <jackson.version>2.15.0</jackson.version>
         <junit.version>5.9.2</junit.version>
         <kafka.version>3.4.0</kafka.version>
-        <kroxy.extension.version>0.1</kroxy.extension.version>
+        <kroxy.extension.version>0.2</kroxy.extension.version>
```

```suggestion
        <kroxy.extension.version>0.2.0</kroxy.extension.version>
```

I think it needs to be like this because that's the version you released.

---

whoops! thanks, let me change that